Description: Workaround for bug #915102 where AC_JAVA_CHECK_JAR picks up
 one of the fop-<subset>.jar instead of fop.jar
Author: Gilles Filippini <pini@debian.org>
Bug-Debian: https://bugs.debian.org/915102
Forwarded: Debian-specific

--- scilab.orig/scilab/configure.ac
+++ scilab/scilab/configure.ac
@@ -1059,7 +1059,9 @@
 
                 ################ Mandatory for graphic_export features #####
                 # XML to PDF/other Translator
-                AC_JAVA_CHECK_JAR([fop],[org.apache.fop.pdf.PDFInfo],[XML to PDF Translator (fop)])
+                #AC_JAVA_CHECK_JAR([fop],[org.apache.fop.pdf.PDFInfo],[XML to PDF Translator (fop)])
+                PACKAGE_JAR_FILE=/usr/share/java/fop.jar
+                ac_java_classpath=$ac_java_classpath:$PACKAGE_JAR_FILE
                 FOP=$PACKAGE_JAR_FILE
                 AC_SUBST(FOP)
 
